docs/tutorial/using-checkbox/installing_checkbox.rst

```diff
@@ -0,0 +1,56 @@
+===================
+Installing checkbox
+===================
+
+In order to install checkbox you are going to need two parts, a ``runtime``
+and a ``frontend``.
+
+First we need to install the ``runtime``::
+
+   $ sudo snap install checkbox22
```

Comment:
That is because you are a barbarian using Arch, and you tried installing Checkbox in a Ubuntu server container! ;)

Joke aside, I was just thinking less characters to type, but you're right, let's just leave sudo.